app/experimenter/nimbus-ui/src/components/PageHome/sortByStatus.ts

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { getStatus } from "../../lib/experiment";
+import { getAllExperiments_experiments } from "../../types/getAllExperiments";
+
+export interface ExperimentCollector {
+  draft: getAllExperiments_experiments[];
+  review: getAllExperiments_experiments[];
+  live: getAllExperiments_experiments[];
+  complete: getAllExperiments_experiments[];
+}
+
+function sortByStatus(experiments: getAllExperiments_experiments[] = []) {
+  return experiments.reduce<ExperimentCollector>(
+    (collector, experiment) => {
+      const status = getStatus(experiment);
+      if (status.live) {
+        collector.live.push(experiment);
+      } else if (status.complete) {
+        collector.complete.push(experiment);
+      } else if (status.review) {
+        collector.review.push(experiment);
+      } else {
+        collector.draft.push(experiment);
```

Comment:
This would also receive "accepted" I think, that's cool?

Reply:
ah good point... i think those should go in reivew